components/Navbar.jsx

```diff
@@ -89,23 +91,23 @@ export default function Navbar() {
                         <li>
                             <Link
```

Comment:
since all links have same functionality can we move all hrefs to an array and map them and just have one link component to minimise code?

Reply:
yepp, will be easier to add a new link in navbar, will do